lib/buildpackutil.py

```diff
@@ -275,25 +277,33 @@ def fix_mono_config_and_get_env(dot_local_location, mono_lib_dir):
         's|/app/vendor/mono/lib/libgdiplus.so|%s|g' % os.path.join(
             mono_lib_dir, 'libgdiplus.so'
         ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
+        os.path.join(_get_mono_path(dot_local_location, mendix_version), 'etc/mono/config'),
     ])
     subprocess.check_call([
         'sed',
         '-i',
         's|/usr/lib/libMonoPosixHelper.so|%s|g' % os.path.join(
             mono_lib_dir, 'libMonoPosixHelper.so'
         ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
+        os.path.join(_get_mono_path(dot_local_location, mendix_version), 'etc/mono/config'),
     ])
     return env
 
 
-def _get_mono_path(dot_local_location):
-    return get_existing_directory_or_raise([
-        os.path.join(dot_local_location, 'mono'),
-        '/usr/local/share/mono-3.10.0',
-        '/tmp/mono',
-    ], 'Mono not found')
+def _get_mono_path(dot_local_location, mendix_version):
+
+    if LooseVersion(mendix_version) < LooseVersion('6.8'):
+        return get_existing_directory_or_raise([
+            os.path.join(dot_local_location, 'mono'),
+            '/opt/mono-3.10.0',
+            '/tmp/mono',
```

Comment:
I think it's nicer to have `/tmp/mono-3.10.0` as path because you know which version of mono is being used without calling the binary.